src/orquestra/quantum/operators/_openfermion_utils/operator_utils.py

```diff
@@ -0,0 +1,79 @@
+#   Copyright 2017 The OpenFermion Developers
```

Comment:
This one should have Zapata licensing as well.
I think we should just run a [morq](https://github.com/zapatacomputing/orquestra-manifest) script for licensing before merging this PR :)

Reply:
On it!